assets/templates/partials/roundel-ns.tmpl

```diff
@@ -0,0 +1,10 @@
+<a
+    href="https://uksa.statisticsauthority.gov.uk/about-the-authority/uk-statistical-system/types-of-official-statistics/"
+    class="national-statistics__link ons-u-ml-s"
+>
+    <img
+    src="https://cdn.ons.gov.uk/assets/images/ons-logo/kitemark/uksa-kitemark.svg"
```

Comment:
If you leave out the scheme it can avoid problems with secure/non-secure content
```suggestion
    src="//cdn.ons.gov.uk/assets/images/ons-logo/kitemark/uksa-kitemark.svg"
```